common/chat/helper.ts

```diff
@@ -90,6 +91,21 @@ const checkIfSubcourseParticipation = async (participants: string[]): Promise<bo
     return result.length > 0;
 };
 
+const getParticipantsForSubcourseGroupChat = async (subcourse: Subcourse, instructor: User) => {
+    const subcourseParticipants = subcourse.subcourse_participants_pupil;
+
+    const participants = await Promise.all(
+        subcourseParticipants.map(async (participant) => {
+            const { pupilId } = participant;
+            const user = await getUser(`pupil/${pupilId}`);
+            return user;
+        })
+    );
+    participants.push(instructor);
```

Comment:
As instructors is `Student[]` their can more than one instructor, are we handling that elsewhere?